src/inc/CombFilterIf.h

```diff
@@ -102,12 +105,8 @@ class CCombFilterIf
 
 private:
     bool            m_bIsInitialized;   //!< internal bool to check whether the init function has been called
-    CCombFilterBase *m_pCCombFilter;    //!< handle of the comb filter
-
-    float           m_fSampleRate;      //!< audio sample rate in Hz
+    // Array of filters: one per channel.
+    std::vector<std::unique_ptr<CCombFilterBase>> filters;
```

Comment:
Compiler does not like "parse issue
<img width="808" alt="Screen Shot 2022-02-16 at 7 17 55 PM" src="https://user-images.githubusercontent.com/69863091/154379754-3f54287d-adba-4c96-92ef-44c1119f58ef.png">
"

Reply:
C++11